api/base/middleware.py

```diff
@@ -180,233 +139,3 @@ def process_response(self, request, response):
             response.content = s.getvalue()
 
         return response
-
-
-class SloanOverrideWaffleMiddleware(WaffleMiddleware):
```

Comment:
So the idea behind sloan was essentially scientific-style A/B testing experiment. This done by overriding our feature flipper Waffle because that was used for A/B testing and controls cookie behavior. This task was difficult because it incorporated custom domains and anonymous users.